packages/frontend/src/components/common/ShareSpaceModal/index.tsx

```diff
@@ -85,12 +85,138 @@ const ShareSpaceModal: FC<ShareSpaceProps> = ({ space, projectUuid }) => {
+                                    (access, index) => (
+                                        <Group key={index} position="apart">
+                                            <Text>
+                                                {access.first_name}{' '}
+                                                {access.last_name}
+                                            </Text>
+                                            <Group>
+                                                <Text>
+                                                    ORG:{' '}
+                                                    {access.organization_role}
+                                                </Text>
+                                                <Text>
+                                                    PROJ:{access.project_role}
+                                                </Text>
+                                            </Group>
+                                        </Group>
+                                    ),
+                                )}
+                            </Stack>
+
+                            <Stack>
+                                <Text fw={500}>Group access</Text>
+                                {space.testing?.groupAccess?.map(
+                                    (access, index) => (
+                                        <Group key={index} position="apart">
+                                            <Text>{access.group_name}</Text>
+                                            <Text>
+                                                SPACE ROLE: {access.space_role}
+                                            </Text>
+                                        </Group>
+                                    ),
+                                )}
+                            </Stack>
+
+                            <Stack>
+                                <Text fw={500}>User access</Text>
+                                {space.testing?.userAccess?.map(
+                                    (access, index) => (
+                                        <Group key={index} position="apart">
+                                            <Text>
+                                                {access.first_name}{' '}
+                                                {access.last_name}
+                                            </Text>
+
+                                            <Text>
+                                                SPACE ROLE: {access.space_role}
+                                            </Text>
+                                        </Group>
+                                    ),
+                                )}
+                            </Stack>
+                            <ShareSpaceUserList
+                                projectUuid={projectUuid}
+                                space={{
+                                    ...space,
+                                    access:
```

Comment:
mutating the data from the new query to see how compatible it is with the expected props.